packages/react-component-library/src/components/NumberInputE/Input.tsx

```diff
@@ -52,7 +52,7 @@ export const Input: React.FC<InputProps> = ({
         data-testid="number-input-input"
         disabled={isDisabled}
         id={id}
-        value={isFinite(value) ? value : ''}
+        value={value || placeholder || ''}
```

Comment:
Hmm it seems a bit odd to pass a `placeholder` to `value` (also for accessibility reasons). Also ,the placeholder can't be deleted using the backspace key.

What about using the `placeholder` attribute with custom styling if needed?

Reply:
Ooh yes much better - I forgot about that native attribute.